stress/src/geneva_exporter.rs

```diff
@@ -0,0 +1,352 @@
+//! Geneva exporter stress test using the generic stream throughput tester
+//!
+//! Run with: cargo run --bin geneva_stream_stress --release -- [concurrency]
+
+/*
+   ## Hardware & OS:
+    Hardware:
+     - CPU: 16 logical processors (8 physical cores), AMD EPYC 7763 @ 2.45 GHz
+     - L1/L2/L3 cache: 512 KB per core
+     - RAM: 64 GB
+     - OS: Ubuntu 6.6 (WSL2 on Windows 11), x86_64
+
+     Using Tokio multi-thread runtime
+    Using standalone mock server at: http://localhost:8080
+    WARNING: Using MockAuth for GenevaConfigClient. This should only be used in tests!
+    Warming up token cache...
+
+    $ cargo run --bin geneva_exporter --release -- multi 500  continuous
+    Testing Geneva Upload with concurrency level: 500
+        Progress: 288092 ops completed (288092 successful, 100.0%) in 5.00s = 57580.46 ops/sec
```

Comment:
It looks like the upload task is more CPU intensive than the I/O. It would be good to check what's the optimal concurrency level for which we get the highest throughput. I believe we would see the throughput plateau after a certain concurrency level. Anything after that would be adding more overhead in terms of memory used etc.